src/test/java/org/takes/facets/auth/PsCookieTest.java

```diff
@@ -51,8 +51,11 @@ public void addsCookieToResponse() throws IOException {
                     new CcPlain(), "foo", 1L
                 ).exit(new RsEmpty(), new Identity.Simple("urn:test:99"))
             ).print(),
-            Matchers.containsString(
-                "Set-Cookie: foo=urn%3Atest%3A99;Path=/;"
+            Matchers.allOf(
+                Matchers.containsString(
+                    "Set-Cookie: foo=urn%3Atest%3A99;Path=/;"
+                ),
+                Matchers.containsString(";HttpOnly")
```

Comment:
@dmzaytsev Now we can incorporate this into the string above - no need for `Matchers.allOf`
```java
Matchers.containsString(
    "Set-Cookie: foo=urn%3Atest%3A99;Path=/;HttpsOnly"
)
```